app/Http/Controllers/InviteKeyController.php

```diff
@@ -4,56 +4,37 @@
 
 use App\Enums\Roles;
 use App\Enums\Statuses;
+use App\Http\Requests\GenerateKeysRequest;
 use App\Models\Game;
 use App\Models\InviteKey;
-use Illuminate\Http\Request;
 
-class ConfigController extends Controller
+class InviteKeyController extends Controller
 {
-    public function index()
+    public function get(InviteKey $key)
     {
-        return view('main_screen', ['games' => Game::all()]);
-    }
-
-    public function storeGame()
-    {
-        $game = Game::create();
-        return redirect()->route('GameScreen', ['id' => $game->id]);
-    }
-
-    public function gameScreen($id)
-    {
-        $police_keys = InviteKey::where('game_id', '=', strval($id))->where('role', '=', Roles::Police)->get();
-        $thief_keys = InviteKey::where('game_id', '=', strval($id))->where('role', '=', Roles::Thief)->get();
-        $game = Game::find($id);
-
-        if (isset($game)) {
-            switch ($game->status) {
-                case Statuses::Config:
-                    return view('config.main', compact(['police_keys', 'thief_keys', 'id']));
-                default:
-                    return view('game.main', compact(['police_keys', 'thief_keys', 'id']));
-            }
+        if ($key->user()->count() > 0) {
+            response()->json([
```

Comment:
Om berichten terug te sturen, gebruikte stond er eerst een methode om specifiek aan te geven dat een error bij de `value` hoorde, zodat error-messages makkelijk bij properties konden worden geplaatst. Door alleen `error` maakt het te lastig om het in te app te herkennen. Maar ik vind het niet erg als dit PR wordt gemerged, want dan kan dit later nog worden toegevoegd.
Deze code ggebruikte ik eerst:
```
CustomErrorService::failedApiResponse('Niet gevonden', [
            'value' => ['De code is onjuist of al in gebruik'],
        ], 404);
```